src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/services/UsersService.java

```diff
@@ -85,5 +85,24 @@ public void follow(Integer customerId, Integer sellerId) {
         customerRepository.save(customer);
     }
 
+    /**
+     * This method is used to trigger unfollowing sequence for a customer unfollowing a seller
+     * @param customerId - customer Id
+     * @param sellerId - seller Id
+     */
+    public void unfollow(Integer customerId, Integer sellerId) {
+        Customer customer = customerRepository.findById(customerId)
+                .orElseThrow(() -> new ResourceNotFoundException("Customer", customerId));
+        Seller seller = sellerRepository.findById(sellerId)
+                .orElseThrow(() -> new ResourceNotFoundException("Seller", sellerId));
+
+        if (!customer.isFollowing(seller))
+            throw new ApiException(HttpStatus.BAD_REQUEST, "already_unfollows_error", "User already unfollows seller");
```

Comment:
Already unfollows is strange, i think customer_isnt_following_error, something like that